frontends/main/src/app-pages/UnitsListingPage/UnitsListingPage.test.tsx

```diff
@@ -138,49 +99,43 @@ describe("DepartmentListingPage", () => {
 
   it("Shows unit properties within the proper section", async () => {
     const { units, courseCounts, programCounts } = setupApis()
+
     renderWithProviders(<UnitsListingPage />)
+
+    const academicSection = screen.getByTestId("UnitSection-academic")
+    const professionalSection = screen.getByTestId("UnitSection-professional")
+
     await waitFor(() => {
-      const academicSection = screen.getByTestId("UnitSection-academic")
-      const professionalSection = screen.getByTestId("UnitSection-professional")
-      units.forEach(async (unit) => {
```

Comment:
This PR is fixing a bug where `/units` page links were broken. This test should have caught that, except it wasn't actually doing anything because `forEach` can't be awaited (it always returns `undefined`) So the test effectively ended on line 144.